src/App.jsx

```diff
@@ -75,5 +84,27 @@ export default function App() {
     throw new Error(`Unhandled authState.type: ${authState.type}`);
   }
 
-  return <NewIssuePage />;
+  return (
+    <ErrorBoundary FallbackComponent={ErrorFallback}>
+      <NewIssuePage />
+    </ErrorBoundary>
+  );
+}
+
+function ErrorFallback({ error /*, resetErrorBoundary */ }) {
+  return (
+    <Box m="4" justifyContent="center">
+      <Heading sx={{ mb: 4 }}>An error occured</Heading>
+      <Text>
+        {error.message}
+        <br />
+        <br />
+        If the problem persists, please let us know on{" "}
+        <Link href="https://github.com/project-forms/project-forms.github.io/issues">
+          GitHub
+        </Link>
+        .
+      </Text>
+    </Box>
```

Comment:
Maybe out of the scope of this task but, do we want to offer users a way to recover from an error (a link to get back to the homepage with all reset so you see the Login form?)